### PR TITLE
Added introduction.md, filled out README.md a little, added link to i…

### PR DESCRIPTION
**Pull request by cafebedouin**
…ntroduction in index.md

Just trying a few things, pulling from the Vue site to make sure my IntelliJ workflow makes sense and this is how we should work together.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -5,24 +5,27 @@ title: Fleet SDK
...
-  - title: "Write once, run anywhere."
-    details: Out-of-the-box compatible with Node.js and Browser environments.
+- title: Simplicity first
+  details: Fleet abstracts all complex tasks and makes heavy use Fluent APIs to deliver a great developer experience.
+- title: Power of Ergo meets JavaScript
+  details: JavaScript allows running off-chain code directly on user's browser, which means better decentralization as it makes backends unnecessary for most cases.
+- title: "Pure JavaScript"
+  details: Enjoy the experience of writing frictionless off-chain code with a fully typed and pure JS library.
+- title: "Write once, run anywhere."
+  details: Out-of-the-box compatible with Node.js and Browser environments.
```
Comment (capt-nemo429) [inline docs/index.md:30]: It looks like your editor messed with `index.md` indentation, can you please check what's possibly wrong?

File: docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+you want to dive right in.
+
+## What is Fleet SDK? 
+
+Fleet SDK (pronounced /flēt/) is a TypeScript framework for building simple or complex applications on the [Ergo 
+blockchain](https://ergoplatform.org/). It builds on top of [Sigma.js](https://github.
+com/ScorexFoundation/sigmastate-interpreter) and provides a declarative and component-based model to
+communicate with Ergo nodes, the ErgoScript interpreter and Ergo protocol implementations.
+
+Here is a minimal example:
```
Comment (capt-nemo429) [inline docs/introduction.md:13]: I think this example is redundant, as we already have it on "Getting Started".

File: docs/introduction.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+This is an overview of Fleet SDK. See [Getting Started](https://fleet-sdk.github.io/docs/guide/getting-started) if 
+you want to dive right in.
+
+## What is Fleet SDK? 
+
+Fleet SDK (pronounced /flēt/) is a TypeScript framework for building simple or complex applications on the [Ergo 
+blockchain](https://ergoplatform.org/). It builds on top of [Sigma.js](https://github.
+com/ScorexFoundation/sigmastate-interpreter) and provides a declarative and component-based model to
+communicate with Ergo nodes, the ErgoScript interpreter and Ergo protocol implementations.
```
Comment (capt-nemo429) [inline docs/introduction.md:11]: 1. Actually Fleet is an off-chain framework
2. Fleet it not entirely build on top of Sigma.JS, only the interpreter and compiler packages are, see: https://github.com/fleet-sdk/fleet#packages
3. Fleet is, in its conception, an off line library, so I think we can remove "communicate with Ergo nodes" part

Maybe worths writing something along these lines: "Feet SDK is an easy-to-use, modular and extensible Ergo Platform's off-chain code SDK (Software Development Kit), written entirely in TypeScript." as in "Getting Started"

**Review by capt-nemo429 — CHANGES_REQUESTED** [review]
Maybe you can remove introduction.md file and add its topics into getting-started file

File: docs/index.md (hunk 1 above)
Comment (cafebedouin) [reply docs/index.md:30]: Yeah, IntelliJ is new to me. I'll fix this and the others tomorrow.

File: docs/introduction.md (hunk 3 above)
Comment (cafebedouin) [reply docs/introduction.md:11]: Yeah, I'm trying to get my head around it, and much of this is new to me. Appreciate any direction when I'm wrong. I apologize for here in the beginning. I might slow you down at first while I get up to speed.

File: docs/index.md (hunk 1 above)
Comment (capt-nemo429) [reply docs/index.md:30]: You can try using Visual Studio Code as well :)

File: docs/introduction.md (hunk 2 above)
Comment (capt-nemo429) [reply docs/introduction.md:13]: Yeah, we can expand later if necessary

File: docs/introduction.md (hunk 3 above)
Comment (capt-nemo429) [reply docs/introduction.md:11]: Don't worry about that, my friend )

**Pull request closed without merging**